src/js/contentscripts/socialwidgets.js

```diff
@@ -330,7 +330,7 @@ function replaceIndividualButton(tracker) {
 *                            replaced
 */
 function getTrackerData(callback) {
-  chrome.runtime.sendMessage({checkReplaceButton:document.location}, function(response) {
+  chrome.runtime.sendMessage({checkReplaceButton:document.location.hostname}, function(response) {
```

Comment:
Should we file a Bugzilla issue for this? New and different behavior from Chrome. Do we need to do a patch release to fix this here (and elsewhere in the code) before Firefox beta lands to stable?